source/renderer/app/containers/tokens/WalletTokenPickerDialogContainer.js

```diff
@@ -0,0 +1,77 @@
+// @flow
+
+import { observer, inject } from 'mobx-react';
+import React, { forwardRef, useCallback, useImperativeHandle } from 'react';
+import type { StatelessFunctionalComponent } from 'react';
+import type { ActionsMap } from '../../actions';
+import type { Props as PickerProps } from '../../components/wallet/tokens/wallet-token-picker/types';
+import WalletTokenPicker from '../../components/wallet/tokens/wallet-token-picker/WalletTokenPicker';
+import type { StoresMap } from '../../stores';
+
+export type Api = {
+  open: () => void,
+};
+
+type Props = PickerProps & {
```

Comment:
Would be possible to make this component stateless by providing `isOpen` and `onOpen` property. WalletSendForm.js could have a property `onOpenTokenPicker` and `WalletSendPage` container invokes `actions.dialogs.open.trigger` on `onOpenTokenPicker`

Reply:
Yep, it is possible, but if we chose the approach of passing down the `open`, `close` and `isOpen` functions then this component is redundant and we can use the `Picker` component directly. I just noticed in our codebase container components dedicated for dialogs and this is why I created this one.